lms/djangoapps/discussion_api/api.py

```diff
@@ -361,19 +366,76 @@ def get_comment_list(request, thread_id, endorsed, page, page_size):
     return get_paginated_data(request, results, page, num_pages)
 
 
+def _check_fields(allowed_fields, data, message):
+    """
+    Checks that the keys given in data is in allowed_fields
+
+    Parameters:
```

Comment:
super-nit: s/Parameters/Arguments, to be consistent with the rest of the edx-platform